docs/websites/nginx/nginx-and-pagespeed-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,132 @@
+---
+author:
+    name: Linode Community
+    email: docs@linode.com
+description: 'Learn how you can install Nginx with the ngx_pagespeed module on Ubuntu 14.04.'
+keywords: 'install ngx_pagespeed, install Nginx from source'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+published: 'Friday, September 11th, 2015'
+modified: Friday, September 11th, 2015
+modified_by:
+    name: Linode
+title: 'How to Install ngx_pagespeed module with Nginx on Ubuntu 14.04'
+contributor:
+    name: Vaibhav Rajput
+    link: https://twitter.com/IamVRajput
+external_resources:
+ - '[Google PageSpeed Filter Docs](https://developers.google.com/speed/pagespeed/module/config_filters)'
+---
+
+This document describes how you can install Nginx (pronounced engine-x) with the ngx_pagespeed module. PageSpeed module helps us to minify CSS, javascript, images, remove whitespaces from HTML and many other features.
+
+Make sure that before starting this guide you have read through and completed our [Getting Started](/docs/getting-started/) guide.
+
+##Set the hostname
+
+Before you install any packages, ensure that your hostname is correct by completing the [Setting Your Hostname](/docs/getting-started#sph_setting-the-hostname) section of the Getting Started guide. Issue the following commands to verify:
+
+    hostname
+    hostname -f
+
+##System Setup
+
+Make sure your system is up to date using apt:
+
+    sudo apt-get update

```

Comment:
We're missing the upgrade portion here. Also, Ubuntu (and Debian 8) now have the plain apt command for added legibility. The two commands could be combined.

`apt update && apt upgrade`

or

`apt-get update && apt-get upgrade`